docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[dataframe-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-wizard-limitations]]
+=== {dataframe-cap} wizard limitation
+
+If you have fields that contain arrays, you cannot select these fields in the source 
+index table in the {kib} {dataframe} wizard. You can, however, use those fields 
+when you group and aggregate your data in the wizard.
+
+[float]
+[[df-ccs-limitations]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
+
+[float]
+[[df-kibana-limitations]]
+=== Displaying {dataframe-transforms} in {kib} limitation
+
+The {kib} *Machine Learning* > *Data Frames* page lists up to 100 {dataframe-transforms}.
+You can work-around this limitation by calling the 
+{ref}/get-data-frame-transform.html[GET {dataframe-transforms} API] 
+with the `size` parameter.
+
+[float]
+[[df-dateformat-limitations]]
+=== Date histogram limitation
+
+When you use the {ref}/data-frame-apis.html[{dataframe-transforms} API] and a date 
+histogram is used as a group by field, the defined interval and the used time format 
+must have the same time fidelity, otherwise it might cause issues in the {dataframe}.
+
+For example, if you set the interval of the date histogram to one minute (1m), 
+then make sure that the time format is `yyyy-MM-dd HH:mm` instead of 
+`yyyy-MM-dd HH:00`.
+
+[float]
+[[df-aggresponse-limitations]]
+=== Strings in aggregation responses
+
+If an aggregation response contains a string (for example, due to a typo in the 
+field name that results in a "NaN" response), the {dataframe} will fail to index 
+the value without warning or causing an error in the {es} logs. However, it results 
+in an empty destination index.
+
+This is a https://github.com/elastic/elasticsearch/issues/43194[known issue] that 
```

Comment:
I feel a little uncomfortable promising that an issue will be fixed--I'm sure it will be but maybe it's better to just say something like this:
```suggestion
See https://github.com/elastic/elasticsearch/issues/43194[Issue #43194].
```